src/music_gui.cpp

```diff
@@ -470,12 +497,37 @@ struct MusicTrackSelectionWindow : public Window {
 				break;
 		}
 	}
+
+	virtual void OnDropdownClose(Point pt, int widget, int index, bool instant_close)
+	{
```

Comment:
I think you wanted to implement OnDropdownSelect.
The default OnDropdownClose method should be fine.